sdk/identity/identity/images/mermaidjs/DefaultAzureCredentialAuthFlow.md

```diff
@@ -5,7 +5,7 @@
 %% 2. Run command: mmdc -i DefaultAzureCredentialAuthFlow.md -o DefaultAzureCredentialAuthFlow.svg
 
 flowchart LR;
-    A(Environment):::deployed ==> B(Managed Identity):::deployed ==> D(Azure CLI):::developer ==> E(Azure PowerShell):::developer;
+    A(Environment):::deployed ==> B(Managed Identity):::deployed ==> C(Azure Developer CLI):::developer ==> D(Azure CLI):::developer ==> E(Azure PowerShell):::developer;
```

Comment:
In the docs links at the bottom of this file, replace "docs.microsoft.com" with "learn.microsoft.com".

Reply:
updated. thanks!